pkg/apis/istio/v1beta1/istio_types.go

```diff
@@ -630,11 +631,12 @@ const (
 type TracingConfiguration struct {
 	Enabled *bool `json:"enabled,omitempty"`
 	// +kubebuilder:validation:Enum=zipkin;lightstep;datadog;stackdriver
-	Tracer       TracerType                `json:"tracer,omitempty"`
-	Zipkin       ZipkinConfiguration       `json:"zipkin,omitempty"`
-	Lightstep    LightstepConfiguration    `json:"lightstep,omitempty"`
-	Datadog      DatadogConfiugration      `json:"datadog,omitempty"`
-	Strackdriver StrackdriverConfiguration `json:"stackdriver,omitempty"`
+	Tracer       TracerType                      `json:"tracer,omitempty"`
+	Zipkin       ZipkinConfiguration             `json:"zipkin,omitempty"`
+	Lightstep    LightstepConfiguration          `json:"lightstep,omitempty"`
+	Datadog      DatadogConfiugration            `json:"datadog,omitempty"`
+	Strackdriver StrackdriverConfiguration       `json:"stackdriver,omitempty"`
+	CustomTags   map[string]apiextensionsv1.JSON `json:"customTags,omitempty"`
```

Comment:
A short comment (one sentence) might be nice here to explain what these tags are for.